xbmc/video/VideoLibraryQueue.h

```diff
@@ -84,7 +85,7 @@ class CVideoLibraryQueue : protected CJobQueue
    \param[in] refreshAll Whether to refresh all sub-items (in case of a tvshow)
    \param[in] searchTitle Title to use for the search (instead of determining it from the item's filename/path)
    */
-  void RefreshItem(CFileItemPtr item, bool ignoreNfo = false, bool forceRefresh = true, bool refreshAll = false, const std::string& searchTitle = "");
+  void RefreshItem(std::shared_ptr<CFileItem> item, bool ignoreNfo = false, bool forceRefresh = true, bool refreshAll = false, const std::string& searchTitle = "");
```

Comment:
I noticed there wasn't a corresponding .cpp file change, and the reason why is because `CFileItemPtr` is still used in the cpp file. Should we made the cpp file consistent with the header? Also it looks like some cleanup/consistency can be done in the cpp file (comparing smart pointer to NULL, std::move optimizations, etc)

Reply:
i did the first part. there are always more cleanups that can be done but that's not on topic for this pr.